github-for-jira.sd.yml

```diff
@@ -367,6 +367,16 @@ environmentOverrides:
               - github-for-jira
             decryptingServices:
               - github-for-jira
+
+      - &cryptorJiraInstanceSecrets
+        name: jira-instance-secrets
+        type: cryptor
+        attributes:
+          parameters:
+            encryptingServices:
+              - github-for-jira
+            decryptingServices:
+              - github-for-jira
```

Comment:
Wait, shouldn't there only be one 'global' one of this?  it's one per service, not one per key right?

Also, shouldn't this be part of the global resources?  Right now it's only on ddev and prod, which means this could break in prod

Reply:
1. Good questions, let me check whether it is per key or per service.
2. Not in global is because the doc recommend dev/stage share the same one.

Reply:
but instead of using the same string as key alias above, it breaks it down into different parts?  seems weird... I'd expect something like `key: micros/github-for-jira/jira-instance-secrets` or something

Reply:
2. but why? what's the advantage?

Reply:
from our perspective, it's less code (and less change of screwing up) the config if we just add all resources in the global

Reply:
> but instead of using the same string as key alias above, it breaks it down into different parts? seems weird... I'd expect something like `key: micros/github-for-jira/jira-instance-secrets` or something

Well, I didn't invent it at the first place. I think the `key` has to be same as the `name` . And why is it broken down? I don't know.

Reply:
> 2. but why? what's the advantage?
This is from the doc.
```
As far as Cryptor is concerned, dev and staging Micros environments both map to the same resource (ie. KMS keys). Cryptor only distinguishes between production and non-production.
```